docs/tutorials/advanced/classification_of_strategies.rst

```diff
@@ -24,7 +24,7 @@ This allows us to, for example, quickly identify all the stochastic
 strategies::
 
     >>> len([s for s in axl.strategies if s().classifier['stochastic']])
-    40
+    41
 
```

Comment:
:+1:

On Thu, Jun 16, 2016 at 1:29 PM Owen Campbell <notifications@github.com>
wrote:

> In docs/tutorials/advanced/classification_of_strategies.rst
> <https://github.com/Axelrod-Python/Axelrod/pull/633#discussion_r67335105>:
>
> > @@ -24,7 +24,7 @@ This allows us to, for example, quickly identify all the stochastic
> >  strategies::
> >
> >      >>> len([s for s in axl.strategies if s().classifier['stochastic']])
> > -    40
> > +    41
> >
>
> Ah yes - I was looking at the memory depth for Naive Prober by mistake
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Axelrod-Python/Axelrod/pull/633/files/907fd35e0288d1ce0927041ec821309516513010#r67335105>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACCGWnwvjl9UK5na1lJiEFj-iZeoEJZoks5qMUGWgaJpZM4I3KSF>
> .
>